dev-haskell/gargoyle-postgresql/files/gargoyle-postgresql-0.2.0.1-add-executables-flag.patch

```diff
@@ -0,0 +1,45 @@
+diff -urN gargoyle-postgresql-0.2.0.1-old/gargoyle-postgresql.cabal gargoyle-postgresql-0.2.0.1/gargoyle-postgresql.cabal
+--- gargoyle-postgresql-0.2.0.1-old/gargoyle-postgresql.cabal	2001-09-08 19:46:40.000000000 -0600
++++ gargoyle-postgresql-0.2.0.1/gargoyle-postgresql.cabal	2021-01-13 23:01:16.396740818 -0700
+@@ -37,9 +37,14 @@
+ 
+ -- Requires postgres to be in the environment to run
+ flag enable-psql-test
++  description: Enable gargoyle-psql-test test-suite
+   default: False
+   manual:  True
+ 
++flag executables
++  description: Install executables
++  default: False
++
+ library
+   exposed-modules:  Gargoyle.PostgreSQL
+   ghc-options:      -Wall
+@@ -72,6 +77,8 @@
+     , process
+     , text
+     , unix
++  if !flag(executables)
++    buildable: False
```

Comment:
Should the library be included in this flag? It seems that if you leave this in, the ebuild wouldn't install anything with `USE="-executables"`

Reply:
There is a client that is installed with this package:
https://hackage.haskell.org/package/gargoyle-postgresql

But, I don't see the use for it, so I have it toggled using the `executables` USE flag, which is disabled by default.

Reply:
Ah, I had misread the patch, my mistake.